test/e2e/applicationautoscaling/tests/test_scalingpolicy.py

```diff
@@ -0,0 +1,86 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Integration tests for the Application Auto Scaling ScalingPolicy API.
+"""
+
+import boto3
+import pytest
+import logging
+from typing import Dict, Tuple
+import time
+
+from applicationautoscaling import SERVICE_NAME, service_marker, CRD_GROUP, CRD_VERSION
+from applicationautoscaling.replacement_values import REPLACEMENT_VALUES
+from applicationautoscaling.bootstrap_resources import TestBootstrapResources, get_bootstrap_resources
+from common.resources import load_resource_file, random_suffix_name
+from common import k8s
+
+RESOURCE_PLURAL = "scalingpolicies"
+
+
+@pytest.fixture(scope="module")
+def applicationautoscaling_client():
+    return boto3.client("application-autoscaling")
+
+@pytest.fixture(scope="module")
+def bootstrap_resources() -> TestBootstrapResources:
+    return get_bootstrap_resources()
+
+@service_marker
+@pytest.mark.canary
+class TestScalingPolicy:
+    def _generate_dynamodb_policy(self, bootstrap_resources: TestBootstrapResources) -> Tuple[k8s.CustomResourceReference, Dict]:
+        resource_name = random_suffix_name("dynamodb-scaling-policy", 32)
+
+        replacements = REPLACEMENT_VALUES.copy()
+        replacements["SCALINGPOLICY_NAME"] = resource_name
+        replacements["DYNAMODB_TABLE"] = bootstrap_resources.ScalableDynamoTableName
```

Comment:
unless I'm mistaken, I think you want this to be:

```python
        replacements["DYNAMODB_TABLE"] = bootstrap_resources.RegisteredDynamoTableName
```

otherwise there's no use of the RegisteredDynamoTableName attribute of bootstrap_resources anywhere.